src/metpy/plots/declarative.py

```diff
@@ -465,13 +500,29 @@ def draw(self):
                 self.layers_edgecolor *= len(self.layers)
             if len(self.layers) > len(self.layers_linewidth):
                 self.layers_linewidth *= len(self.layers)
+            if len(self.layers) > len(self.layers_linestyle):
+                self.layers_linestyle *= len(self.layers)
+            if len(self.layers) > len(self.layers_zorder):
+                self.layers_zorder *= len(self.layers)
+            if len(self.layers) > len(self.layers_alpha):
+                self.layers_alpha *= len(self.layers)
             for i, feat in enumerate(self._layer_features):
                 if self.layers[i] in ['', 'land', 'lake', 'river']:
```

Comment:
There is a bug here that I identified while working on this PR. The only feature that is impacted by this line is 'land'. I don't think 'river' should be in this list since it doesn't use a face color as a line object and 'lake' should be 'lakes' and for consistency this would mean adding 'ocean'. This would break a couple of tests as a result of a change. The other option is just to remove 'lake' and 'river' since they are not doing anything at current.

Reply:
👍 to removing `'river'` if it really doesn't do anything. I thought maybe they filled at higher scales?

Fine to change to `'lakes'` and `'ocean'` for consistency. In my ideal world, maybe we can tweak the tests to avoid needing new images?

Reply:
Okay, so after a little investigating we have the conundrum of whether to allow the setting of an edgecolor for the fill features of `'land'`, `'ocean'`, and `'lakes'`. In the current version of declarative, we were intending to set the edgecolor to `'face'` for these types of features, which would preclude an individual from being able to set a different edgecolor. No amount of tweaking the tests would allow them to pass with this being the case.

Upon reflection I think we should have our list of using the `'face'` edgecolor for `'land'` and `'ocean'` and not allow the user to specify as there would always be some other feature that could be (and most likely would be) plotted (e.g., borders, coastline) to get the "finished" edge. I think we should leave `'lakes'` off the list, as there would be more circumstances where you might want to have a separate edgecolor from facecolor when plotting that feature. This would result in only one test needing an updated figure in the test suite and the visual difference is minimal, meaning end users would likely not even notice the slight change in behavior.